app/src/main/java/com/example/todolistproject/model/Task.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.todolistproject.model
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "Task")
+data class Task(
+
+    @PrimaryKey(autoGenerate = true)
+    var id: Long?,
+
+    @ColumnInfo(name = "isCompleted")
+    var isCompleted: Boolean? = false,
+
+    @ColumnInfo(name = "title")
+    var title: String,
+
+    @ColumnInfo(name = "description")
+    var description: String? = "",
+
+    @ColumnInfo(name = "startDate")
+    var startDate: String?,
+
+    @ColumnInfo(name = "endDate")
+    var endDate: String?,
```

Comment:
```suggestion
    @PrimaryKey(autoGenerate = true)
    var id: Long? = null,

    @ColumnInfo(name = "isCompleted")
    var isCompleted: Boolean? = false,

    @ColumnInfo(name = "title")
    var title: String = "",

    @ColumnInfo(name = "description")
    var description: String? = "",

    @ColumnInfo(name = "startDate")
    var startDate: String? = "",

    @ColumnInfo(name = "endDate")
    var endDate: String? = "",
```
p3; 기본 생성자를 오버라이드 하지 않고 위와 같이 기본 값들을 넣어주는건 어떨까요?
이렇게 수정하면 

TaskActivity 105라인의 
val task =
            Task(
                id = null,
                isCompleted = false,
                title = title,
                description = description,
                startDate = startDate,
                endDate = endDate
            )
를
val task =
            Task(
                title = title,
                description = description,
                startDate = startDate,
                endDate = endDate
            )
            이렇게 사용할 수 있어요.